Source/SpatialGDK/Private/SpatialInterop.cpp

```diff
@@ -70,13 +72,18 @@ void USpatialInterop::Init(USpatialOS* Instance, USpatialNetDriver* Driver, FTim
 		{
 			LinkExistingSingletonActors(*op.Update.singleton_name_to_entity_id().data());
 		}
+		if (op.Update.stably_named_path_to_entity_id().data())
+		{
+			DeleteIrrelevantReplicatedStablyNamedActors(*op.Update.stably_named_path_to_entity_id().data());
```

Comment:
Remove newline

Reply:
Which one do you mean? On my screen I only see
```
if(check())
{
 Delete...
}
```

Reply:
sorry it is my github view that trolled me.